<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCaptureAuxiliary</class>
 <widget class="QWidget" name="CameraCaptureAuxiliary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择摄像头</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CameraCaptureAuxiliary{
background-color: rgb(211, 211, 211);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(82, 204, 144);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16px &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>选择摄像头</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_close">
        <property name="minimumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Login/img/Login/close_green_Click.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWdg" native="true">
     <property name="styleSheet">
      <string notr="true">#bottomWdg{
background-color: rgb(255, 255, 255);
border-bottom: 1px solid rgb(196, 196, 196);
border-left: 1px solid rgb(196, 196, 196);
border-right: 1px solid rgb(196, 196, 196);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_render" native="true">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>135</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>135</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/ctrl/img/ctrl/media_setting_01.jpg);
border-radius : 4px; 

</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="cmbcamera">
          <property name="minimumSize">
           <size>
            <width>278</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>278</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QComboBox {
font: 14px &quot;微软雅黑&quot;;
    border: 1px solid rgb(221, 221, 221);
    border-radius: 3px;
    padding: 1px 1px 1px 5px;
    background-color: rgb(255, 255, 255);
	color: rgb(68, 68, 68);
}

QComboBox::down-arrow {
	image: url(:/ctrl/img/ctrl/media_setting_down.png);

}
QComboBox::drop-down{
    border:0px;
}

QComboBox::up-arrow {
	image: url(:/ctrl/img/ctrl/media_setting_up.png);
}

QComboBox QAbstractItemView{
    outline: 0px;
    border-radius:4px;
	color: rgb(68, 68, 68);
	selection-background-color: rgb(82, 204, 144);
	selection-radius:4px;
	background-color: rgb(255, 255, 255);
	border-left:15px;
 border: 1px solid rgb(221, 221, 221);
}

/*滚动条样式 开始*/
QScrollBar:vertical
{
border-radius:4px;
width:6px;
background:rgba(255,255,255, 255);
margin:0px,0px,0px,0px;

}

QScrollBar::handle:vertical
{
background:rgba(210,210,210, 255);
border-radius:3px;
min-height:20;
border: 1px solid #D2D2D2;
}

QComboBox:!enabled {
    background-color: rgb(232, 232, 232);
    border: 1px solid rgb(210, 210, 210);
	color: rgb(153, 153, 153);
}
</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_apply">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
font: 14px &quot;微软雅黑&quot;;
background-color: rgb(82, 204, 144);
border: 0px;
border-radius: 4px;
color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color: rgb(99, 224, 162);
}

QPushButton:pressed{
background-color: rgb(52, 184, 119);
}

QPushButton:!enabled{
background-color: rgba(82, 204, 144,51);
}</string>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
